Neos.Flow/Configuration/Objects.yaml

```diff
@@ -250,6 +250,21 @@ Neos\Flow\Monitor\FileMonitor:
 # HTTP                                                                     #
 #                                                                          #
 
+Psr\Http\Message\ServerRequestFactoryInterface:
+  className: Tuupola\Http\Factory\ServerRequestFactory
```

Comment:
We could just provide our own implementations to be independent. The code is not too much.

Reply:
Yep, need to look into it. I haven't used factories in my change so far, but probably makes sense.